apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/anyAssertions.kt

```diff
@@ -30,22 +30,56 @@ fun <T> Expect<T>.notToBe(expected: T): Expect<T> = _logicAppend { notToBe(expec
 /**
  * Expects that the subject of the assertion is the same instance as [expected].
  *
+ * Deprecated as atrium moves to a consistent ‘to + infinitive’ naming convention. Use [toBeTheSameAs] instead.
+ * This function will be removed in version 1.0.0. See
+ * [atrium-roadmap#93](https://github.com/robstoll/atrium-roadmap/issues/93) for details and to give feedback.
+ *
  * @return An [Expect] for the current subject of the assertion.
  * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ */
+@Deprecated(
+    "Replaced by toBeTheSameAs. Will be removed with version 1.0.0",
+    ReplaceWith("this.toBeTheSameAs<T>(expected)")
+) // TODO remove with 1.0.0
+fun <T> Expect<T>.isSameAs(expected: T): Expect<T> = toBeTheSameAs(expected)
+
+/**
+ * Expects that the subject of `this` assertion is the same instance as [expected].
  *
- * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.AnyAssertionSamples.isSameAs
+ * @return an [Expect] for the current subject of `this` assertion.
+ * @throws AssertionError if the subject of `this` assertion is not the same instance as [expected].
```

Comment:
Might needs to be in the text as it depends on the context whether this function throws or not. I am also happy with removing the `@throws AssertionError` from all functions since this adds little to the KDoc

Reply:
I think the doc either needs to state clearly when the exception is thrown, or not at all. ‘Might’ makes me mad because I’d like to know _when_.

Proposal:

> if the subject of `this` assertion is not the same instance as [expected] and this function is not called from inside an assertion creator lambda. If the call happens from inside an assertion creator lambda, the exception will be thrown by the function receiving the lambda.

But I am also completely fine with just removing all the `@throws`. I just think that if they are there, they need to be clear.

Reply:
So I suggest to remove them, I guess most people will expect that it throws an AssertionError and if in doubt, it is easy to find out if required. Less cluttering the code with something which has only little gain is a code thing IMO.

Btw. it is hard to come up with something which is actually true in all cases. 

> if the subject of this assertion is not the same instance as [expected] and this function is not called from inside an assertion creator lambda. If the call happens from inside an assertion creator lambda, the exception will be thrown by the function receiving the lambda. But only if it is not contributed to an ExplanatoryAssertionGroup or similar because such AssertionGroups usually do not cause an Exception... unless someone has explicitly marked them to fail. Yet, it still depends on the Expect if it is going to throw or not, in case of the default implementation, it is actually only the RootExpect which will throw... but you never know :laughing: 

And that's really not helpful either for a user -- I guess it is kind of helpful for a core contributor, to better understand when an exception is thrown but should probably be documented elsewhere.

Reply:
I suggest we remove them also after the merge, created a task in the project:
https://github.com/robstoll/atrium/projects/1#column-12080551